Address/FI.ts

```diff
@@ -12,4 +12,7 @@ export namespace FI {
 			typeof(value.city) == "string" &&
 			value.countryCode == "FI"
 	}
+	export function create(): FI {
+		return { countryCode: "FI", street: "", zipCode: "", city: ""}
```

Comment:
If there are no reasons, don't deviate from the "natural" order when using the properties. It makes it easier to make changes and to ensure everything is there.